app/packages/registration-schema/validations.ts

```diff
@@ -65,6 +65,40 @@ export function buildValidation(groups: SchemaBlockGroup[], node?: NodeModel) {
     return ret;
 }
 
+export function validateNodeLicense() {
+    return async (_: unknown, __: unknown, ___: unknown, changes: DraftRegistration, content: DraftRegistration) => {
+        let validateLicenseTarget = await content.license;
+        let validateNodeLicenseTarget = content.nodeLicense;
+        if (changes.license) {
+            validateLicenseTarget = changes.license;
+        }
+        if (changes.nodeLicense) {
+            validateNodeLicenseTarget = changes.nodeLicense;
+        }
+        if (validateLicenseTarget.get('requiredFields').length === 0) {
+            return true;
+        }
+        const missingFieldsList: string[] = [];
+        for (const item of validateLicenseTarget.get('requiredFields')) {
+            if (!validateNodeLicenseTarget![item]) {
```

Comment:
I maybe be wrong but I think there might be a case where both `validateLicenseTarget` and `validateNodeLicenseTarget` are null since we are force validating??